content/zh-cn/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -1,271 +1,310 @@
-                <p>In order for the new Deployment to be accessible without using the proxy, a Service is required which will be explained in the next modules.</p>
-                -->
-                <p>你可以运行以下命令通过代理的 API 访问 Pod：</p>
-                <p><b><code>curl http://localhost:8001/api/v1/namespaces/default/pods/$POD_NAME/</code></b></p>
-                <p>为了不使用代理也能访问新的 Deployment，需要一个 Service，这将在下一个模块中讲述。</p>
-            </div>
-
-      </div>
-      <div class="row">
-          <p>
-            <!--
-            Once you're ready, move on to <a href="/docs/tutorials/kubernetes-basics/explore/explore-intro/" title="Viewing Pods and Nodes">Viewing Pods and Nodes</a>.</p>
-            -->
-            如果你准备好了，请访问<a href="/zh-cn/docs/tutorials/kubernetes-basics/explore/explore-intro/" title="查看 Pod 和 Node">查看 Pod 和 Node</a>。</p>
-          </p>
-      </div>
-
-    </main>
-
-</div>
-
-</body>
-</html>
+---
+title: 使用 kubectl 创建 Deployment
+weight: 10
+description: |-
+  学习应用的部署。
+  使用 kubectl 在 Kubernetes 上部署第一个应用。
+---
+<!--
+title: Using kubectl to Create a Deployment
+weight: 10
+description: |-
+  Learn about application Deployments.
+  Deploy your first app on Kubernetes with kubectl.
+-->
+
+<!DOCTYPE html>
+
+<html lang="zh">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+         <div class="col-md-8">
+          <!-- <h3>Objectives</h3> -->
+          <h3>目标</h3>
+                <!-- <ul>
+                    <li>Learn about application Deployments.</li>
+                    <li>Deploy your first app on Kubernetes with kubectl.</li>
+                </ul> -->
+                <ul>
+                  <li>学习应用的部署。</li>
+                  <li>使用 kubectl 在 Kubernetes 上部署第一个应用。</li>
+              </ul>
+            </div>
+
+            <div class="col-md-8">
+                <!-- <h3>Kubernetes Deployments</h3> -->
+                <h3>Kubernetes 部署</h3>
+                <!-- <p>
+                Once you have a <a href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/">
+                running Kubernetes cluster</a>, you can deploy your containerized applications on top of it.
+                To do so, you create a Kubernetes <b>Deployment</b>. The Deployment instructs Kubernetes
+                how to create and update instances of your application. Once you've created a Deployment, the Kubernetes
+                control plane schedules the application instances included in that Deployment to run on
+                individual Nodes in the cluster.
+                </p> -->
+                <p>
+                一旦<a href="/zh-cn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/">运行了 Kubernetes 集群</a>，
+                就可以在其上部署容器化应用程序。为此，你需要创建 Kubernetes <b>Deployment</b>。
```

Comment:
```suggestion
                就可以在其上部署容器化应用。为此，你需要创建 Kubernetes <b>Deployment</b>。
```
`应用` 会不会更顺口一些，目前`应用程序`有 412 个，`应用`有 1200+ 个